layer/lambda_utils/lambda_utils/logging.py

```diff
@@ -207,3 +223,17 @@ def wrapper(*args: P.args, logger: Logger = None, **kwargs: P.kwargs) -> RT:
         return wrapper
 
     return decorator
+
+
+def make_scoped_log_action(
+    scope_fn,
+    *args,
+    **kwargs,
+):
+    """
+    Uses a partial to define the request scoped values once, rather than each
+    @log_action.
+    """
+    _kwargs: dict[str, Any] = {**kwargs}
+    _kwargs["scope_fn"] = scope_fn
+    return partial(log_action, *args, **_kwargs)
```

Comment:
Finally worked out the function signature forwarding:

```suggestion
def make_scoped_log_action(
    scope_fn, _: Callable[P, RT] = log_action
) -> Callable[P, RT]:
    """
    Uses a partial to define the request scoped values once, rather than each
    @log_action.
    """
    return partial(log_action, scope_fn=scope_fn)
```